Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+sounddevice = "*"
+scipy = "*"
+numpy = "*"
+rx = "*"
+keyboard = "*"
+mouse = "*"
+opencv-python = "*"
+ffmpeg = "*"
+soundfile = "*"
+tensorflow = ">=2.3.1, <2.4.0"
+matplotlib = "*"
+python-language-server = {extras = ["all"], version = "*"}
```

Comment:
Why do we need this in the project?
I think such things should be installed separately, not with user distribution of the package.

Reply:
Sorry, I meant to install it as a dev dependency